store/tikv/mock_tikv_service.go

```diff
@@ -0,0 +1,205 @@
+package tikv
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/kvproto/pkg/coprocessor"
+	"github.com/pingcap/kvproto/pkg/kvrpcpb"
+	"github.com/pingcap/kvproto/pkg/tikvpb"
+	"github.com/pingcap/tidb/util/logutil"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+)
+
+type server struct{}
+
+func (s *server) KvGet(context.Context, *kvrpcpb.GetRequest) (*kvrpcpb.GetResponse, error) {
```

Comment:
We should not implement a function with a `unimplement` panic. The better way is to define `server` as `type server struct{InterfaceYouWantToImplement}`.